assets/section-image-banner.css

```diff
@@ -335,18 +330,13 @@
 @media screen and (min-width: 750px) {
   .banner--desktop-transparent .banner__box {
     background: transparent;
-    --color-foreground: 255, 255, 255;
-    --color-button: 255, 255, 255;
-    --color-button-text: 0, 0, 0;
     max-width: 89rem;
     border: none;
     border-radius: 0;
     box-shadow: none;
   }
 
   .banner--desktop-transparent .button--secondary {
-    --color-button: 255, 255, 255;
-    --color-button-text: 255, 255, 255;
     --alpha-button-background: 0;
```

Comment:
Although I obviously know why this is happening, this is a weird result. Or at least it's different than the desktop equivalent. I'd expect the secondary button to stay transparent unless the mobile container was enabled https://screenshot.click/19-41-z63lp-yb7ol.mp4

Reply:
I pushed 1c9af8e, which makes this work again. I _think_ it's fine to have the background be transparent in all cases like that, but let me know if you notice anything unexpected.

Reply:
hm, i worked for me locally, but it's not working here 😅🤔

Reply:
I pulled your latest changes from this PR and works on my local computer. I guess some magic is happening with connection between github and our demo store. If I go to the editor and check the code there I can see that CSS file hasn't been updated 
<img width="1168" alt="Screenshot 2023-05-25 at 11 05 03 AM" src="https://github.com/Shopify/dawn/assets/105315663/c233dcdf-80c1-4ab3-8b0f-0d56ff542552">

Reply:
I reconnected the editor and updated the link in the description to the PR. Now it looks everything working good.